frontend/public/components/utils/breadcrumbs.ts

```diff
@@ -1,11 +1,23 @@
 import { K8sKind } from '../../module/k8s';
+import { LAST_NAMESPACE_NAME_LOCAL_STORAGE_KEY } from '@console/shared/src/constants';
+import { ALL_NAMESPACES_KEY } from '@console/shared/src/constants/common';
+
+
+export const formatBreadcrumbPath = (match: any, customPlural?: string) => {
+  const lastNamespace = localStorage.getItem(LAST_NAMESPACE_NAME_LOCAL_STORAGE_KEY);
+    if (match.params.ns) {
+        return lastNamespace === ALL_NAMESPACES_KEY
+      ? `/k8s/all-namespaces/${customPlural ? customPlural : match.params.plural}`
+      : `/k8s/ns/${lastNamespace !== null ? lastNamespace : match.params.ns }/${customPlural ? customPlural : match.params.plural}`;
```

Comment:
If last namespace isn't all namespaces, you want to always use `match.params.ns`.

```suggestion
      : `/k8s/ns/${match.params.ns}/${customPlural || match.params.plural}`;
```

Reply:
Thanks, I'll fix it.